packages/core/src/features/facets/data/facet_additional_data.json

```diff
@@ -467,8 +467,6 @@
     "facet_filter": "family_histories.relationship_age_at_diagnosis",
     "facet_type": "age",
     "label": "Relationship Age at Diagnosis",
-    "minimum": 0,
-    "maximum": 89,
```

Comment:
is this no longer needed? what happens when we start getting data for this property? will it display a number range card with min 0 and max 89?

Reply:
I'll make a note of a better fix in the spike about improving this area. thanks!